tests/Functional/Bundle/Maker/MakeFactoryTest.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Zenstruck\Foundry\Tests\Functional\Bundle\Maker;
 
+use Composer\InstalledVersions;
+use Composer\Semver\VersionParser;
```

Comment:
Are these required? They don't appear to be used.

Reply:
indeed